domains/auth/Auth.API/Tests/AuthControllerTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using API.Controllers;
+using API.Models;
+using API.Services.OidcProviders;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+using Xunit.Categories;
+
+namespace Tests;
+
+[UnitTest]
+public class AuthControllerTest
```

Comment:
```suggestion
public class TestAuthController
```


This will be using the same convention as the other tests in this Tests-project.

Reply:
I will also suggest to move the location of this file, so it matches the project being tested (that is "API" in this case).

Reply:
fixed